File: cosas_c/pares.c
#include <stdio.h>

int main(void) {

	int numero;

	printf("Escribe un numero: ");
	scanf("%d", &numero);

	if (numero % 2 == 0) {
			printf("El es par %i", numero);

	} else {
			printf("El numero no es par %i", numero);

	}

	return 0;
}

File: ejemplos_python/pares.py
#!/usr/bin/python3


def main():
    LIM = 51
    con = 1

    while con < LIM:
        if con % 2 == 0:
            print(f"El numero {con} es par")
        con += 1



if __name__=="__main__":
    main()
